cypress/integration/pages/tag-creator.spec.js

```diff
@@ -4,8 +4,8 @@ describe('Tag Generator Page (Tag Creator)', () => {
   const CHANGE_AFFILIATED_ORGANIZATION = 'Hack for LA';
   const AFFILIATED_TEST_URL = 'codeforboston / voiceapp311';
   const AFFILIATED_TEST_TAGS = ['code-for-boston', 'code-for-america', 'alexa', 'trash'];
-  const AFFILIATED_NEW_TAGS = ['tag1','code-for-all', 'civictechindex'];
-  const UNAFFILIATED_NEW_TAGS = ['tag1','tag2', 'tag3'];
+  const UNAFFILIATED_NEW_TAGS = ['tag1', 'tag2', 'tag3'];
+  const expectedAffiliatedNewTags = ['civictechindex', 'code-for-all', 'tag1'];
```

Comment:
Could expectedAffiliatedNewTags be updated to AFFILIATED_NEW_TAGS ?

Reply:
expectedAffiliatedNewTags makes sense because along with the new tag from the user other tags come from parent org tags and auto-generated "civictechindex" tag